plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/controller-role-bindings.yaml

```diff
@@ -204,6 +204,23 @@ items:
   - kind: ServiceAccount
     name: job-controller
     namespace: kube-system
+- apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
what is this separate yaml file for? we already have bootstrappolicy/controller_policy.go code to define the role/binding.

Reply:
This is auto generated